### PR TITLE
Implement attendee UI

### PR DESCRIPTION
**Pull request by jbmichel14**
To test the attendee UI, the boolean in MainActivity has to be set to true.

### CODE REVIEW

File: app/src/androidTest/java/com/github/dedis/student20_pop/ui/ConnectFragmentTest.java [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (AlexisTabin) [inline app/src/androidTest/java/com/github/dedis/student20_pop/ui/ConnectFragmentTest.java:1]: useless line

**Review by AlexisTabin — APPROVED** [review]

File: app/src/main/java/com/github/dedis/student20_pop/ui/AttendeeFragment.java [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+import com.github.dedis.student20_pop.utility.attendeeUI.ExpandableListViewEventAdapter;
+import com.github.dedis.student20_pop.model.Event;
+import com.github.dedis.student20_pop.model.Lao;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+
+//How do we pass the info from Connect UI to Attendee UI ?
```
Comment (celinecamacho) [inline app/src/main/java/com/github/dedis/student20_pop/ui/AttendeeFragment.java:25]: Are these comments relevant?

File: app/src/main/java/com/github/dedis/student20_pop/ui/AttendeeFragment.java [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+        expandableListView.setAdapter(listViewEventAdapter);
+        expandableListView.expandGroup(0);
+        expandableListView.expandGroup(1);
+
+        //Display Properties
+        View properties = rootView.findViewById(R.id.properties_view);
+        ((TextView) properties.findViewById(R.id.organization_name)).setText(lao.getName());
+        ((TextView) properties.findViewById(R.id.witness_list)).setText("Witnesses: [id, id, id]");
+        //later: witness list, other information?
+        //((TextView) properties.findViewById(R.id.witness_list)).setText(lao.getWitnesses().toString());
```
Comment (celinecamacho) [inline app/src/main/java/com/github/dedis/student20_pop/ui/AttendeeFragment.java:61]: Is this comment important?

File: app/src/main/java/com/github/dedis/student20_pop/utility/attendeeUI/ExpandableListViewEventAdapter.java [hunk 4]
```diff
@@ -0,0 +1,264 @@
...
+    }
+
+    @Override
+    public long getGroupId(int groupPosition) {
+        return groupPosition;
+    }
+
+    @Override
+    public long getChildId(int groupPosition, int childPosition) {
+        return childPosition; //((Event) getChild(groupPosition, childPosition)).getUid();
```
Comment (celinecamacho) [inline app/src/main/java/com/github/dedis/student20_pop/utility/attendeeUI/ExpandableListViewEventAdapter.java:115]: Is this comment important?

File: app/src/main/java/com/github/dedis/student20_pop/utility/attendeeUI/package-info.java [hunk 5]
```diff
@@ -0,0 +1,7 @@
+/**
+ * Classes used to display the lhe elements in relation to the attendee UI.
```
Comment (celinecamacho) [inline app/src/main/java/com/github/dedis/student20_pop/utility/attendeeUI/package-info.java:2]: There is a typo between "the" and "elements"

File: app/src/main/res/layout/fragment_organizer_poll.xml [hunk 6]
```diff
@@ -0,0 +1,189 @@
...
+                    android:layout_height="wrap_content"
+                    android:text="@string/now"
+                    app:layout_constraintStart_toEndOf="@+id/start_time_title"
+                    app:layout_constraintTop_toTopOf="parent" />
+
+                <EditText
+                    android:id="@+id/start_date_edit_text"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:ems="10"
```
Comment (celinecamacho) [inline app/src/main/res/layout/fragment_organizer_poll.xml:68]: The 10 value should be stored in the file res/values/dimens.xml, same thing for line 79, 105 and 115

File: app/src/androidTest/java/com/github/dedis/student20_pop/MainActivityTest.java [hunk 7]
```diff
@@ -55,4 +56,19 @@ public void onClickCancelLAOTest() {
         onView(withId(R.id.button_cancel_launch)).perform(click());
         onView(withId(R.id.fragment_home)).check(matches(isDisplayed()));
     }
+
+    //These tests only pass when testingAttendee is true in MainActivity
```
Comment (celinecamacho) [inline app/src/androidTest/java/com/github/dedis/student20_pop/MainActivityTest.java:60]: You can create a function in MainActivity to set testingAttendee to true at the beginning of each test

**Review by celinecamacho — CHANGES_REQUESTED** [review]
Good job! There's just a method in MainActivity that could be useful to have and some comments left hanging

**Review by celinecamacho — APPROVED** [review]

**Pull request merged**